packages/features/ee/organizations/lib/orgDomains.ts

```diff
@@ -43,3 +43,7 @@ export function subdomainSuffix() {
   const urlSplit = WEBAPP_URL.replace("https://", "")?.replace("http://", "").split(".");
   return urlSplit.length === 3 ? urlSplit.slice(1).join(".") : urlSplit.join(".");
```

Comment:
Though it's existing code, I think we shouldn't give special treatment to 3 dots. We have already add an issue where someone was hosting with more than 3 dot domains.
https://github.com/calcom/cal.com/pull/9803